scripts/plaintext2bloom.js

```diff
@@ -1,21 +1,38 @@
+
+var path = require('path');
 var fs = require('fs');
 // do an npm i bloomfilter first
 var bf = require('bloomfilter');
 
-var bloom = new bf.BloomFilter(
-  10 * 54000, // number of bits to allocate.
-  8        // number of hash functions.
-);
-fs.readFile('human-ans-shorter.txt', 'utf8', function (err,data) {
+var inputFilename = process.argv[2];
+var outputFilename = process.argv[3];
+
+if (! (inputFilename && outputFilename)) {
+  console.error('usage: node plaintext2bloom.js <input filename> <output filename>');
+  return 1;
+}
+
+fs.readFile(inputFilename, 'utf8', function (err, data) {
   if (err) {
     return console.log(err);
   }
+
   var s = data.split('\n');
+  var bloom = new bf.BloomFilter(
+    10 * s.length,  // number of bits to allocate per item
```

Comment:
Instead of hard coding the number of entries, base the size on the length of the dictionary.